src/document/DocumentAnnotator.ts

```diff
@@ -55,6 +56,7 @@ export default class DocumentAnnotator extends BaseAnnotator {
         this.annotatedEl.classList.add(CLASS_ANNOTATIONS_LOADED);
 
         this.render();
+        this.setLayerVisibility(getVisibility(this.store.getState()));
```

Comment:
Does this have to run after `render` or can it be be in the super class init?

Reply:
> Does this have to run after `render` or can it be be in the super class init?

Yeah it has to run after `render` cause only after render, can we select elements and set their style to `visibility: hidden`.